### PR TITLE
fix: build seed docker multiplatform

### PR DESCRIPTION
**Pull request by armandobelardo**
Ensure seed containers are built to run on diff platforms

### CODE REVIEW

File: .github/workflows/build-seed.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - ab/fixSeed
```
Comment (dsinghvi) [inline .github/workflows/build-seed.yml:7]: lets take this out before merging!

Comment (armandobelardo) [reply .github/workflows/build-seed.yml:7]: good catch! removed!

**Comment by dsinghvi** [comment]
@armandobelardo it took 49m to build the seed container for python, should we just not make it multiplatform?

**Comment by armandobelardo** [comment]
> @armandobelardo it took 49m to build the seed container for python, should we just not make it multiplatform?

Yeah it's horrible, nothing I tried made it faster though

When it's not multiplatform we just can't run it locally, the container that's used for seed in Actions is different from the one we use on macs, so we need to build for both archs

We could change the base runners for the actions though so architectures match, I don't have much of a preference since this image shouldn't change

**Comment by dsinghvi** [comment]
> We could change the base runners for the actions though so architectures match, I don't have much of a preference since this image shouldn't change

Let's do this, 49m on the Seed Runner is quite expensive for Github actions

**Review by dsinghvi — APPROVED** [review]

**Pull request closed without merging**